promotion/suggestions.go

```diff
@@ -216,6 +217,21 @@ func (service *Service) Suggest(ctx context.Context, credentials []CredentialBin
 		return err
 	}
 
+	// Delete this section once the issue is completed
+	// https://github.com/brave-intl/bat-go/issues/263
+	if suggestion.OrderID != nil {
+		// TODO Should we iterate through the funding types like below and create transactions for each type?
+		// Is this a good place to put this or should we moved this into the RunNextSuggestion function?
+		_, err := service.datastore.CreateTransaction(*suggestion.OrderID, eventMap["id"].(string), "completed", "BAT", "virtual-grant", total)
+		if err != nil {
+			return errors.Wrap(err, "Error recording order transaction")
```

Comment:
I would suggest we not use `github.com/pkg/errors` anymore as the stdlib has mechanisms for dealing with wrapping errors now for causation.  The use of `errors` in both stdlib and in this package makes it confusing if we are using regular stdlib errors or this package's error... if you try to "Unwrap" the result of this result it will not work, as in this package you need to use the causer interface.  Also mentioning because this library has gone into maintenance mode as well.

Reply:
Would you create an issue for this refactoring? 😄

Reply:
https://github.com/brave-intl/bat-go/issues/276